stanford-earth/SE3_BLT
# EARTH-290: Enable stanford_paragraph_quote_banner.

# READY FOR REVIEW

# Summary
- Enables the Quote banner paragraph type

# Needed By (Date)
- End of sprint

# Urgency
- Normal

# Steps to Test

1. Check out EARTH-290 on SE3_BLT, stanford_paragraphs, and stanford_components
2. Run `drush cim`, `drush csim local`, `drush features-import-all`
3. Go to a complex_page type and create a new Quote Banner paragraph
4. Review paragraph display

# Associated Issues and/or People
- EARTH-290

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
